index.html

```diff
@@ -0,0 +1,200 @@
+<!DOCTYPE html>
+<html lang="en-US">
+<head>
+<meta charset="utf-8" />
+<title>Web of Things (WoT) Discovery</title>
+<script class="remove"
+    src="https://www.w3.org/Tools/respec/respec-w3c-common"></script>
+<script class="remove">
+        var respecConfig = {
+        lint: {
+            "check-punctuation": true,
+            "local-refs-exist": true,
+            "no-http-props": true,
+            "no-headingless-sections": true
+        },
+        doJsonLd : true,
+        specStatus : "ED",
+        shortName : "wot-discovery",
+        copyrightStart : 2017,
+        wg : "Web of Things Working Group",
+        wgURI : "https://www.w3.org/WoT/WG/",
+        wgPatentURI : "https://www.w3.org/2004/01/pp-impl/95969/status",
+        wgPublicList : "public-wot-wg",
+        github: {
+            repoURL: "https://github.com/w3c/wot-discovery",
+            branch: "master"
+        },
+        // previousPublishDate: "2019-05-16",
+        // previousMaturity: "CR",
+        editors : [ {
+            name : "Michael McCool",
+            w3cid : "93137",
+            company : "Intel Corp.",
+            companyURL : "https://www.intel.com/"
+        } ],
+        otherLinks : [
+                {
+                    key : "Contributors",
+                    data : [ {
+                        value : "In the GitHub repository",
+                        href : "https://github.com/w3c/wot-architecture/graphs/contributors"
+                    } ]
+                }],
+        localBiblio : {
+           /// none yet
+        }
+    };
+</script>
+<style type="text/css">
+a[href].internalDFN {
+        color: inherit;
+        border-bottom: 1px solid #99c;
+        text-decoration: none;
+}
+img.wot-diagram {
+    max-width: 90%;
+    height: auto;
+}
+</style>
+</head>
+<body>
+    <section id="abstract">
+        <p>The W3C Web of Things (WoT) is intended to enable
+            interoperability across IoT platforms and application
+            domains.
+            Overall, the goal of the WoT is to preserve and complement existing
+            IoT standards and solutions. In general, the W3C WoT architecture is 
+            designed to describe what exists rather than to prescribe
+            what to implement.
```

Comment:
FWIW I've never really agreed with this statement from the architecture document, but it seems particularly contradictory to include it in the abstract of a specification which it appears will then go on to describe a Directory Service API which WoT clients and servers should implement.

Reply:
Yeah, that was just cut-and-paste.   I'm going to rewrite this section.